llvm/lib/CodeGen/ScheduleDAGInstrs.cpp

```diff
@@ -633,8 +637,6 @@ std::optional<unsigned> ScheduleDAGInstrs::initSUnit(MachineInstr &MI) {
 
 void ScheduleDAGInstrs::initSUnits() {
   ScheduleDAG::clearDAG();
```

Comment:
nit: this looks at odds with the clearDAG in `ScheduleDAGInstrs::enterRegion`

Reply:
I could remove that one now, I just thought it does not hurt. But I agree it does look strange